src/helpers/issue.ts

```diff
@@ -118,7 +118,8 @@ export const getAllIssueComments = async (issue_number: number): Promise<Comment
 
       await checkRateLimitGit(response?.headers);
 
-      if (response.data) {
+      // Fixing infinite loop here, it keeps looping even when its an empty array
+      if (response.data && response.data.length > 0) {
```

Comment:
```suggestion
      if (response?.data?.length > 0) {
```

Could be more concise.

Reply:
> Could be more concise.

Yea, i guess that'd be more consise.. Thanks